README.md

```diff
@@ -49,6 +50,12 @@ cordova platform update android
 cordova platform update ios
 ```
 
+## Deprecating old versions
+
+Since CodePush is migrating to a new service all versions of cordova-plugin-code-push that lower than **[1.12.0](https://github.com/microsoft/cordova-plugin-code-push/releases/tag/v1.12.0)** will not work in the nearest future.
```

Comment:
```suggestion
Since CodePush is migrating to a new service all versions of cordova-plugin-code-push lower than **[1.12.0](https://github.com/microsoft/cordova-plugin-code-push/releases/tag/v1.12.0)** will not work in the nearest future.
```